tokio/src/runtime/metrics/batch.rs

```diff
@@ -279,3 +287,13 @@ cfg_rt_multi_thread! {
 pub(crate) fn duration_as_u64(dur: Duration) -> u64 {
     u64::try_from(dur.as_nanos()).unwrap_or(u64::MAX)
 }
+
+/// Gate time metrics as platforms like `WASM` do not typically support [`std::time`]
+/// <https://github.com/tokio-rs/tokio/issues/7319>
+fn now() -> Option<Instant> {
+    if cfg!(target_family = "wasm") && !cfg!(feature = "time") {
```

Comment:
You should do an exact match: `cfg! (all(target_arch = "wasm", target_os = "unknown", target_vendor = "unknown"))`. Instant::now should be available just fine on all other Wasm targets that have std.